invenio_app_ils/cli.py

```diff
@@ -373,3 +375,21 @@ def data(n_docs, n_items, n_loans, n_keywords):
 
     click.secho('Now indexing...', fg='green')
     indexer.process_bulk_queue()
+
+
+@click.group()
+def patrons():
+    """Patrons data CLI."""
+
+@patrons.command()
+@with_appcontext
+def index():
+    """Index patrons."""
+    patrons  = User.query.all()
+    indexer = PatronsIndexer()
+
+    click.secho('Now indexing {0} patrons'.format(len(patrons)),  fg='green')
+
+    for pat in patrons:
+        patron = Patron(pat.id)
+        indexer.index(patron)
```

Comment:
It wasn't possible to bulk index here because users are not records, right?

Reply:
Yes, the indexer for Patrons has the bare minimum functionality in order to index synchronously. We will address it properly in `invenio-indexer` so we can enable parametrized behaviour of the record class.